eng/Versions.props

```diff
@@ -30,11 +30,11 @@
     <MicrosoftDotNetBuildTasksPackagingVersion>1.0.0-beta.18565.8</MicrosoftDotNetBuildTasksPackagingVersion>
```

Comment:
Should we remove these redundant properties now?

Reply:
good call